### DISCUSSION TITLE
Step by Step guide for sending log using filebeat to custom index in Elasticsearch

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#5705] [Unsupported Versions] [answered]
Hello, I am new to all of this SO, Elasticsearch, Logstash, Filebeat. ALL of it. I am trying to consolidate all of the different logging systems we have into a single source such as SO.
I need to be able to send our tomcat logs from various apps to their own index. App:A -> indexA, AppB-> IndexB.
In Elasticsearch I have created an index using the ES API which after a reboot appeared to delete the index.

I am failing to understand the need for Logstash if Elastcisearch is now doing all of the parsings.
I have read multiple articles that referencing using Logstash to create the index within Elasticsearch, but with SO things don't seem to line up..

Could someone point me in the direction of a straightforward way to creating an index that will be searchable from Kibana and Grafana in SO
Sending the log file from a Centos7 box using Filebeat

SO version 2.3.70

**Comment by weslambert** [comment, marked as answer]
You should be able to use the Filebeat module for Tomcat to help with this:

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-tomcat.html

You would configure the module as you would normally via the Elastic-provided instructions, except for loading the pipeline(s) -- this is already done by Security Onion.
